Source/WebKit/Shared/Cocoa/WKNSDictionary.mm

```diff
@@ -68,7 +68,7 @@ - (id)objectForKey:(id)key
     if (!exists)
         return nil;
 
-    return value ? value->wrapper() : [NSNull null];
+    return value ? (id)value->wrapper() : [NSNull null];
```

Comment:
I am surprised that the cast to `id` is needed. What kind of error do you get without it?

Reply:
Same as above with `WKNSArray.mm`.